presto-docs/src/main/sphinx/functions/ml.rst

```diff
@@ -0,0 +1,151 @@
+===================
+Machine Learning Functions
+===================
+
+The machine learning plugin in Presto provides machine learning functionality
+as an aggregation function. It enables us to train Support Vector Machine (SVM)
+based classifier and regressor for the supervised learning problems.
```

Comment:
should it be "classifiers and regressors" or is an article missing earlier in this sentence?